README.md

```diff
@@ -86,6 +86,11 @@ This file can be used to integrate with CI systems.
 gotestsum --junitfile unit-tests.xml
 ```
 
+If the package names in the `testsuite.name` or `testcase.classname` fields do not
+work well with your CI system the values can be modified using one of the
+flags: `--junitfile-testsuite-name-format`, `--junitfile-testcase-classname-format`.
```

Comment:
I think possible values (and differences between them) need to be documented as well.
Basically the: `"full, relative, short"`